fortio_main.go

```diff
@@ -94,6 +95,15 @@ func main() {
 		usage("Unable to extract percentiles from -p: ", err)
 	}
 
+	// Following code determines whether the t flag has been provided together with the endless flag as they are
+	// both mutual exclusive. durationFlag can't be tested because if t flag is omitted the durationFlag will hold
+	// the default value.
```

Comment:
it's nice but that seems like a lot of code just to have an error, maybe it's fine to just have endless overwrite the duration
(or we could test if the duration isn't the default anymore)